ftw/news/viewlets/news_date.py

```diff
@@ -0,0 +1,21 @@
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from ftw.news.contents.news import INewsSchema
+from plone.app.layout.viewlets import ViewletBase
+import datetime
+
+
+class NewsDateViewlet(ViewletBase):
+
+    template = ViewPageTemplateFile('news_date.pt')
+
+    def render(self):
+        return self.template()
+
+    def news_date(self):
```

Comment:
The template is calling this method twice (`tal:condition` and `tal:content`), which takes unnecessary time and can be optimized.
I'd move calculate the date in an `update` method (which should automatically be called by the viewlet manager) and store the news-date as attribute on self.